ksql-functional-tests/src/test/java/io/confluent/ksql/test/loader/ExpectedTopologiesTestLoader.java

```diff
@@ -222,7 +215,7 @@ public static TopologyAndConfigs readTopologyFile(
     final InputStream s = ExpectedTopologiesTestLoader.class.getClassLoader()
```

Comment:
Does the stream get closed anywhere? (In a finally preferably)

Reply:
Yes, by the try with resource block underneath